docs/src/app/versions.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Unless I'm misunderstanding, this is going to import whatever version of React is current when the `versions.jsx` js is built.

It does make the rather large assumption that it's possible to have two scripts (`versions.js` and `app.js`) that are running different versions of React loaded concurrently, and I hadn't researched whether that was realistic. It seems it isn't by default, but there is a hack to enable this: https://github.com/facebook/react/issues/1939#issuecomment-101790804

The alternative might be to not import React at all in `versions.js`, and to depend on the version of React loaded by any particular version of the Material-UI demo-site. This assumes that the DropDown and the parts of Material-UI it uses are compatible with the earliest version of React in use.

If neither of those is possible, then the only other option (that uses Material-UI) would be to go back and edit the source for each prior version to add the dropdown using the version of  Drop Down Menu, and therefore React, that was current at the time.

I'd rather avoid that if possible (editing the source of all the prior versions), so a native `<select>` (styled to look as close to Material-UI) and with plain js is the only other option I can think of.